guides/source/routing.md

```diff
@@ -142,6 +142,8 @@ Creating a resourceful route will also expose a number of helpers to the control
 
 Each of these helpers has a corresponding `_url` helper (such as `photos_url`) which returns the same path prefixed with the current host, port, and path prefix.
 
+TIP: You can generate a full list of paths and helper names with `rake routes`.
```

Comment:
Docs elsewhere are using `rails routes`, can you use that here too?

Reply:
Yes you're right, I am out of date. I'm now aware this is mentioned elsewhere on the page, so will update my PR